server/models/Profile.js

```diff
@@ -18,8 +18,8 @@ const profileSchema = new mongoose.Schema({
     ref: "User",
     required: true,
   },
-  userPhotoUrl: {
-    type: String,
+  userPhotosUrls: {
```

Comment:
Hey @mayanksuchde, are we planning to give the user an option to have single/multiple profile images?

Reply:
I saw this image from the pdf and thought that was the case.

<img width="826" alt="Screen Shot 2021-09-16 at 1 37 07 PM" src="https://user-images.githubusercontent.com/6592064/133659023-32d91bb5-02c7-4512-a84d-04f7e0bbb1af.png">
c.

Reply:
maybe we can choose a single index from the array and put it as a profile pic and the rest can be under the description.
Thoughts??

Reply:
I would suggest we have a separate field for `profilePhoto` (single) and another field for `galleryImages` (multiple).

Reply:
In the above case I've mentioned, we may need to make changes to the UI/create another component which we can use to upload gallery images.

Reply:
> I would suggest we have a separate field for `profilePhoto` (single) and another field for `galleryImages` (multiple).

So in that case I would have to add 
2 separate routes and controllers; one to add profile image and one to add gallery Images, right?

Reply:
and data type would be a String for a profile pic and array for galleryImages?

Reply:
@rajivtitus Hey Rajiv I have made the changes in the model and added profilePic:String and galleryPics:[String] to the model. Will work on  routes and controller